File: mobilne/AndroidStudio/Licznik/app/src/main/java/com/example/licznik/MainActivity.java
package com.example.licznik;

import android.os.Bundle;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    Button btnPolicz;
    TextView textView;
    EditText editText;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btnPolicz = findViewById(R.id.btnLicz);
        textView = findViewById(R.id.textView);
        editText = findViewById(R.id.editText1);

        btnPolicz.setOnClickListener(view -> {
            String text = editText.getText().toString().trim();

            if(text.isEmpty()) {
                textView.setText("0");
                return;
            }

            var tab = text.split("\\s+");
            textView.setText(String.valueOf(tab.length));
        });
    }
}

File: mobilne/AndroidStudio/Aplikacja1/app/src/main/java/com/example/aplikacja1/MainActivity.kt
package com.example.aplikacja1

import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)

        val poleTekstowe = findViewById<EditText>(R.id.poleTekstowe)
        val przycisk = findViewById<Button>(R.id.przycisk)
        val komunikat = findViewById<TextView>(R.id.Komunikat)

        przycisk.setOnClickListener {
            komunikat.text = poleTekstowe.text.toString()
        }
    }
}